Changelog.md

```diff
@@ -116,6 +116,7 @@ Bugfixes:
  * Type Checker: Fix freeze for negative fixed-point literals very close to ``0``, such as ``-1e-100``.
  * Type Checker: Dynamic types as key for public mappings return error instead of assertion fail.
  * Type Checker: Fix internal error when array index value is too large.
+ * Type Checker: Search for a name suggestion when the name is not found after argument-dependant lookup.
```

Comment:
There's a tiny typo. Should be ``argument-dependent``

Reply:
Fixed. Thanks